onnxruntime/core/framework/utils.cc

```diff
@@ -518,6 +546,69 @@ common::Status ExecuteGraph(const SessionState& session_state,
   return status;
 }
 
+#ifdef ENABLE_TRAINING
+common::Status ExecutePartialGraph(const SessionState& session_state, FeedsFetchesManager& feeds_fetches_manager,
+                                   const std::vector<OrtValue>& feeds, std::vector<OrtValue>& fetches, const bool& terminate_flag,
```

Comment:
>const bool&  [](http://example.com/codeflow?start=103&length=12)

could be just bool

Reply:
if we still want to leverage the feature of "terminate_flag", you need to pass by non-const "bool&" 
otherwise, we don't need this flag at all.

Reply:
It was added as protection against infinite loops (primarily for WinML). Whether you need that in a training setup or not I don't know. 

It should be const though - it is only set by the InferenceSession::Run caller (and not internally by us). The executor is just reading that value.

Reply:
I don't see the usage so will remove it.